test/cli/delegated_routing_v1_http_proxy_test.go

```diff
@@ -60,8 +60,12 @@ func TestRoutingV1Proxy(t *testing.T) {
 		nodes := setupNodes(t)
 
 		cidStr := nodes[0].IPFSAddStr(testutils.RandomStr(1000))
-
-		res := nodes[1].IPFS("routing", "findprovs", cidStr)
+		// Reprovide as initialProviderDelay still ongoing
+		res := nodes[0].IPFS("bitswap", "reprovide")
+		if res.Err != nil {
+			t.Fatal(res.Err)
+		}
```

Comment:
Minor: use `require` here and in other tests.
```suggestion
		require.NoError(t, res.Err)
```